public/app/plugins/panel/timeseries/migrations.ts

```diff
@@ -296,7 +296,7 @@ export function flotToGraphOptions(angular: any): { fieldConfig: FieldConfigSour
     graph.fillOpacity = angular.fillGradient * 10; // fill is 0-10
   }
 
-  graph.spanNulls = angular.nullPointMode === NullValueMode.Null;
+  graph.spanNulls = angular.nullPointMode === 'connected';
```

Comment:
NullValueMode.Ignore is 'connected' any reason why you didn't use that instead?

Reply:
Ah, just by mistake, haven't checked the enum. Good catch, @zoltanbedi!